### ISSUE TITLE
Link when passed only onClick should have onKeyDown also

### ISSUE THREAD

**Issue by ElaLukaszewicz** [brainly/style-guide#2033]
When Link has no href and we pass onClick, the link should also have implemented onKeyDown which fires a function when 'Enter' or space is clicked
We also should provide tabIndex to text

**Comment by coderitual** [comment]
@katarzynatobis I believe this issue might not be the case after the last improvement, is that right?

**Comment by katarzynatobis** [comment]
Yes, it's fixed. But you still need to provide `as="button"`

**Comment by coderitual** [comment]
Sure, we are closing it then
cc @ElaLukaszewicz
